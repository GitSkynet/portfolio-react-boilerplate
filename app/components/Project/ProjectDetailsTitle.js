import styled from 'styled-components';

const ProjectDetailsTitle = styled.p`
  margin-top: .25em;
`;

export default ProjectDetailsTitle;
