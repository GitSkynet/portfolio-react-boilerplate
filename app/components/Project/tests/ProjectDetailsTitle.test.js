import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import 'jest-styled-components';

import ProjectDetailsTitle from '../ProjectDetailsTitle';

describe('<ProjectDetailsTitle />', () => {
  it('should render and match the snapshot', () => {
    const wrapper = shallow(<ProjectDetailsTitle />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
