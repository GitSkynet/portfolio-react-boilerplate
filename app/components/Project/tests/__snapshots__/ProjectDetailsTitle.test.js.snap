// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProjectDetailsTitle /> should render and match the snapshot 1`] = `
.c0 {
  margin-top: 0.25em;
}

<p
  className="c0"
/>
`;
